movie-browser/src/App.js

```diff
@@ -28,9 +29,9 @@ toggleMovieSection = () => {
     return (
       <div>
         <Header />
-        <Featured movies={movies} />
-        <MostViewed movies={movies} />
-        <Movies movies={movies} toggleMovieSection={this.toggleMovieSection} shouldShowAllMovies={this.state.shouldShowAllMovies} />
+        <Featured featuredMovies={movies} />
```

Comment:
if `Featured` only render one movie why are u passing the whole thing?

Reply:
`Featued` renders 6 movies, but do you mean I should do this:
```
<Featured featuredMovies={movies.splice(0,6)}
```
?

Reply:
u should pass only the movie's it's going to render :)